plugins/network-elements/nuage-vsp/src/com/cloud/network/resource/NuageVspRequestWrapper.java

```diff
@@ -0,0 +1,78 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package com.cloud.network.resource;
+
+import java.util.Hashtable;
+import java.util.Set;
+
+import org.reflections.Reflections;
+
+import com.cloud.agent.api.Answer;
+import com.cloud.agent.api.Command;
+import com.cloud.resource.CommandWrapper;
+import com.cloud.resource.RequestWrapper;
+import com.cloud.resource.ServerResource;
+
+public class NuageVspRequestWrapper extends RequestWrapper {
+
+    private static NuageVspRequestWrapper instance;
+
+    static {
+        instance = new NuageVspRequestWrapper();
+    }
+
+    Reflections baseWrappers = new Reflections("com.cloud.network.vsp.resource.wrapper");
```

Comment:
Why isn't this attribute declared `private static final`?  My understanding of reflections, it only needs to instantiated once to scan a package path.  It seems unnecessary/expensive to spin it up every time `NuageVspRequestWrapper` is instantiated.